tdm-fe/src/app/components/progress/StatusTimeline.tsx

```diff
@@ -0,0 +1,84 @@
+import '~/app/components/progress/scss/StatusTimeline.scss';
+import ErrorOutlineIcon from '@mui/icons-material/ErrorOutline';
+import TimelineConnector from '@mui/lab/TimelineConnector';
+import TimelineContent from '@mui/lab/TimelineContent';
+import TimelineDot from '@mui/lab/TimelineDot';
+import TimelineItem from '@mui/lab/TimelineItem';
+import TimelineSeparator from '@mui/lab/TimelineSeparator';
+import CircularProgress from '@mui/material/CircularProgress';
+import { useMemo } from 'react';
+import type { TimelineContentProps } from '@mui/lab/TimelineContent';
+import type { TimelineDotProps } from '@mui/lab/TimelineDot';
+
+export type StatusTimeline = {
```

Comment:
<!-- TODO's are small, trivial, but necessary changes. Distinguishing todo comments from issues: or suggestions: helps direct the reader's attention to comments requiring more involvement.  -->
**todo:** rename it to `StatusTimelineProps`‏